mlir/test/Dialect/MIOpen/lowering_threadwise_copy_v2.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: miopen-opt -miopen-lowering-step4 %s | FileCheck %s
+// RUN: miopen-opt -miopen-lowering-step3 -miopen-lowering-step4 %s | FileCheck %s
```

Comment:
Do we still need step4 here now the lowering for these ops have been moved to step3?

Reply:
Yeah, since the tests check for raw_buffer_store

Reply:
Would you rather I rework the tests?

Reply:
That could be a fairly good idea as existing tests are too loose.